rbi/annotations/parse-cron.rbi

```diff
@@ -71,8 +69,6 @@ end
 
 # internal "mutable" time representation
 class CronParser::InternalTime
-  extend T::Generic
```

Comment:
See https://github.com/siebertm/parse-cron/blob/master/lib/cron_parser.rb (not generic)